cmd/kwil-cli/cmds/utils/parse.go

```diff
@@ -0,0 +1,60 @@
+package utils
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+
+	"github.com/kwilteam/kwil-db/cmd/common/display"
+	"github.com/kwilteam/kwil-db/parse"
+	"github.com/spf13/cobra"
+)
+
+func newParseCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "parse <file_path>",
+		Short: "Parse a Kuneiform schema",
+		Long:  `Parse a Kuneiform schema and output the JSON schema.`,
+		Args:  cobra.ExactArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			if args[0] == "" {
+				return display.PrintErr(cmd, fmt.Errorf("file path is required"))
+			}
+
+			file, err := os.ReadFile(args[0])
+			if err != nil {
+				return display.PrintErr(cmd, err)
+			}
+
+			res, err := parse.ParseKuneiform(string(file))
+			if err != nil {
+				return display.PrintErr(cmd, err)
+			}
+
```

Comment:
No, we check the `res.Err()` when we format. Known parsing errors (syntax/type errors) should get returned, but should not brick the whole parser. We format them so that consumers can highlight it.